teams/inspec.md

```diff
@@ -0,0 +1,30 @@
+# InSpec Team
+
+The InSpec team is responsible for InSpec and several resource packs and plugins.
+
+### Project Owners
+
+Clinton Wolfe
+  - Github: [clintoncwolfe](https://github.com/clintoncwolfe)
+  - Slack: @cwolfe
+
+Miah Johnson
+  - Github: [miah](https://github.com/miah)
+  - Slack: @miah
+
+### Project Approvers
+
+### Project Reviewers
+
+## Project Repositories
+
+[inspec](https://github.com/inspec/inspec)
+[inspec-gcp](https://github.com/inspec/inspec-gcp)
+[inspec-digital-ocean](https://github.com/inspec/inspec-digital-ocean)
+[inspec-habitat](https://github.com/inspec/inspec-habitat)
+[inspec-oneview](https://github.com/inspec/inspec-habitat)
+[inspec-vmware](https://github.com/inspec/inspec-vmware)
+[kitchen-inspec](https://github.com/inspec/kitchen-inspec)
+[train-aws](https://github.com/inspec/train-aws)
+[train-habitat](https://github.com/inspec/train-habitat)
+[train-digitalocean](https://github.com/inspec/train-digitalocean)
```

Comment:
I noticed that this list gets rendered as [an impenetrable paragraph of links](https://github.com/chef/chef-oss-practices/blob/8af910f612cb5d239d692015edbaa104e31ea0ed/teams/inspec.md#project-repositories). Instead, would [a bulleted list](https://github.com/chef/chef-oss-practices/blob/9f34cc83254f3676f0dd626b3f62b653fbe3c7cb/teams/supermarket.md#project-repositories) work better?

Reply:
I'm going to merge this and fix it in a larger PR